css/main.css

```diff
@@ -130,6 +139,7 @@ header + section p {
 }
 .selling-points div {
     margin: 0 20px;
+    width: 33%;
```

Comment:
Without a width specified flexbox will only make sure that the objects all fit within one row. The width call forces them to display as three equal items. Attached are screenshots of how it displays without that call.
![screen shot 2016-01-20 at 9 44 59 pm](https://cloud.githubusercontent.com/assets/850942/12472623/1518afda-bfbf-11e5-9333-407ef15886b6.png)
![screen shot 2016-01-20 at 9 44 50 pm](https://cloud.githubusercontent.com/assets/850942/12472624/151adbca-bfbf-11e5-8f60-aaea0bef890a.png)